flags.go

```diff
@@ -50,10 +50,15 @@ var (
 		Name:  "tlsverify",
 		Usage: "Use TLS and verify the remote",
 	}
+	flOverCommit = cli.IntFlag{
+		Name:  "overcommit, oc",
+		Usage: "OverCommit to apply on resouces",
```

Comment:
It actually isn't really a overcommit flag after all, since you're no longer padding the resources but actually assigning the final resources to schedule with the ratio. If I put in 50 I'm actually under instead of overcommitting. 

I wonder if instead of putting in a 105 value, we should just let the user put in 5, and this extra 5% is the overcommit resources so we don't ever have the possibility to go under.

Then in the comment it can say "percentage of container resources to overcommit"